lib/Mojolicious/Guides/Tutorial.pod

```diff
@@ -0,0 +1,938 @@
+
+=encoding utf8
+
+=head1 NAME
+
+Mojolicious::Guides::Tutorial - Get started with Mojolicious
+
+=head1 TUTORIAL
+
+A quick example-driven introduction to the wonders of L<Mojolicious::Lite>. Almost everything you'll learn here also
+applies to full L<Mojolicious> applications.
+
+This is only the first of the L<Mojolicious::Guides>. Other guides delve deeper into topics like
+L<growing|Mojolicious::Guides::Growing> a L<Mojolicious::Lite> prototype into a well-structured L<Mojolicious>
+application, L<routing|Mojolicious::Guides::Routing>, L<rendering|Mojolicious::Guides::Rendering> and more. It is
+highly encouraged that readers continue on to the remaining guides after reading this one.
+
+=head2 Hello World
```

Comment:
Another example that I remember learning late and being shocked by is

```
$ mojo daemon
```

Sometimes you just want a Hello World app for testing connectivity -- and before learning / realizing that `mojo daemon` without a specified app defaults to HelloWorld.pm, I used to `generate lite_app` to run `mojo myapp daemon` for the same purpose.

Not the best example, but showing a Hello World using ojo would be nice here, too.
```
$ perl -Mojo -E 'a({text => time})->start' daemon
```